### PR TITLE
BugFixes and Enhancements

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
1. Added "GitHub - Create Issue Comment" playbook
2. Added "GitHub - Add Repository Collaborator" playbook
3. Added PR Title as issue comment in "Create Pull Request Playbook"
4. Pass input record to child playbook in "Fetch Latest Changes" playbook
5. Added base branch param to "GitHub - Clone Repository" playbook
6. Updated "Get User Confirmation" step name to "Confirm Apply Latest Changes" in "Pull Latest Content from GitHub" playbook
7. passed base branch param to GitHub - Clone Repository" from "Push (Initial) Production Environment Content", "Push Change to Source Control", "Push Content.", "Save FortiSOAR Settings", "Setup Development Environment" and ""Pull Latest Content from GitHub" playbooks
8. Removed unwanted "Update Import Job" and "Modify Configuration Option" steps from "Setup Development Environment" playbook
9. Added "Add Repository Collaborator" steps in "Set GitHub Username" playbook

### CODE REVIEW

File: playbooks/02 - Use Case - CICD/GitHub - Create Issue Comment.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        "commit_message"
+    ],
+    "synchronous": false,
+    "collection": "\/api\/3\/workflow_collections\/4c58fed3-9e16-451a-9528-0be2b71acda4",
+    "versions": [],
+    "triggerStep": "\/api\/3\/workflow_steps\/da0edd7f-6f64-46be-be72-cafe8f4aff07",
+    "steps": [
+        {
+            "@type": "WorkflowStep",
+            "name": "Add Issue Commit",
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/GitHub - Create Issue Comment.json]: Add a Comment to an Issue

File: playbooks/02 - Use Case - CICD/GitHub - Create Issue Comment.json [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                    "org": "{{vars.input.params['org_name']}}",
+                    "body": "{{vars.input.params['commit_message']}}",
+                    "repo": "{{vars.input.params['repo_name']}}",
+                    "repo_type": "Organization",
+                    "issue_number": "{{vars.input.params['cr_number']}}"
+                },
+                "version": "1.0.1",
+                "connector": "github",
+                "operation": "create_issue_comment",
+                "operationTitle": "Create Issue Commnet",
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/GitHub - Create Issue Comment.json]: This title has a typo. Please correct in the connector info.json as well

File: playbooks/02 - Use Case - CICD/GitHub - Create Issue Comment.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/GitHub - Create Issue Comment.json]: done

File: playbooks/02 - Use Case - CICD/GitHub - Create Issue Comment.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/GitHub - Create Issue Comment.json]: done

**Review by cs-pooja-singh — APPROVED** [review]

**Pull request merged**